AddQtAndroidApk.cmake

```diff
@@ -89,12 +88,16 @@ macro(add_qt_android_apk TARGET SOURCE_TARGET)
         set(QT_ANDROID_APP_PACKAGE_NAME org.qtproject.${SOURCE_TARGET})
     endif()
 
-    # set the Android SDK build-tools revision
-    if(ARG_BUILDTOOLS_REVISION)
-        set(QT_ANDROID_SDK_BUILDTOOLS_REVISION ${ARG_BUILDTOOLS_REVISION})
-    else()
-        set(QT_ANDROID_SDK_BUILDTOOLS_REVISION "")
-    endif()
+    # Detect latest Android SDK build-tools revision
+    set(QT_ANDROID_SDK_BUILDTOOLS_REVISION "0.0.0")
+    file(GLOB children RELATIVE ${QT_ANDROID_SDK_ROOT}/build-tools ${QT_ANDROID_SDK_ROOT}/build-tools/*)
+    foreach(child ${children})
+        # find subfolder with greatest version
+        if (${child} VERSION_GREATER ${QT_ANDROID_SDK_BUILDTOOLS_REVISION})
+            set(QT_ANDROID_SDK_BUILDTOOLS_REVISION ${child})
+        endif()
+    endforeach()
+    message("Detected QT_ANDROID_SDK_BUILDTOOLS_REVISION ${QT_ANDROID_SDK_BUILDTOOLS_REVISION}")
```

Comment:
Maybe "Detected Android SDK build tools version ${...}" would be more user friendly, and more consistent with the other similar messages.

Reply:
Fixed.